### PR TITLE
batch process onchain btc withdrawals

### PR DESCRIPTION
**Pull request by djpnewton**
and use our own unconfirmed utxos

### CODE REVIEW

**Review by eoliveros — COMMENTED** [review]
got the following in the logs when using backend to send:
```
[wallet INFO] required: 1414000 sats, sats onchain: 2648181 sats
[task_manager ERROR] exception in task (deposits, withdrawals, orders): 'txid'
[ln_wallet_endpoint INFO] preparing withdrawal with outputs: [{'2NGE3syfgYfgBoaRZUzNZo6xc9VEqgo1jg5': '0.0001sat'}]
[ln_wallet_endpoint INFO] preparing withdrawal with outputs: [{'2NGE3syfgYfgBoaRZUzNZo6xc9VEqgo1jg5': '0.0001btc'}]
[ln_wallet_endpoint INFO] preparing withdrawal with outputs: [{'2NGE3syfgYfgBoaRZUzNZo6xc9VEqgo1jg5': '0.0001btc'}]
[ln_wallet_endpoint INFO] preparing withdrawal with outputs: [{'2NGE3syfgYfgBoaRZUzNZo6xc9VEqgo1jg5': '0.0001sat'}]
[ln_wallet_endpoint INFO] preparing withdrawal with outputs: [{'2NGE3syfgYfgBoaRZUzNZo6xc9VEqgo1jg5': '0.0001btc'}]
[ln_wallet_endpoint INFO] preparing withdrawal with outputs: [{'2NGE3syfgYfgBoaRZUzNZo6xc9VEqgo1jg5': '0.0001btc'}]
[wallet INFO] searching for input tx 25ad15f45a7b236cb7dd4f892d991459e3b36acbb6d91fbca2d04aa47e3a0ea4
```

will try again after i clear all my utxo.... just noticed that this is still draft

File: src/depwith.py [hunk 1]
```diff
@@ -450,6 +451,12 @@ def _crypto_withdrawal_update(crypto_withdrawal: BalanceUpdate):
             updated_records.append(ftx)
         return updated_records
     if crypto_withdrawal.status == crypto_withdrawal.STATUS_AUTHORIZED:
+
+        # we process btc onchain withdrawals as a batch so skip this
+        if crypto_withdrawal.asset == assets.BTC.symbol and crypto_withdrawal.l2_network is None:
+            logger.error('authorized BTC onchain withdrawals will be done in the batch process')
```
Comment (eoliveros) [inline src/depwith.py:457]: is this supposed to be logger.error or logger.info?

**Comment by eoliveros** [comment]
also deployed on my test env and using my test FE sending to 2 different btc address and got the following result according to the transaction:
image

when clicking the txid:
image

Below is when im looking at the utxo:
image

**Review by 4855062503 — APPROVED** [review]
LGTM.

File: src/depwith.py (hunk 1 above)
Comment (djpnewton) [reply src/depwith.py]: good catch, fixed

**Pull request merged**